crates/tabby/src/main.rs

```diff
@@ -45,7 +45,7 @@ pub enum Commands {
     /// Run scheduler progress for cron jobs integrating external code repositories.
     Scheduler(SchedulerArgs),
 
-    /// Manage local models
+    /// Install, list, and delete models for local use
```

Comment:
I suggest add a block of document here to explain a concept - model id.
it can be form of [registry]/name, and being used in many place of `model` sub command.